parachain/pallets/omni-account/src/lib.rs

```diff
@@ -194,14 +215,42 @@ pub mod pallet {
 			auth_type: Option<OmniAccountAuthType>,
 			result: DispatchResult,
 		},
-		/// Intent is requested
-		IntentRequested { who: T::AccountId, intent: Intent },
-		/// Intent is executed
-		IntentExecuted { who: T::AccountId, intent: Intent, result: IntentExecutionResult },
 		/// Member permission set
 		AccountPermissionsSet { who: T::AccountId, member_account_hash: H256 },
 		/// An auth token is requested
 		AuthTokenRequested { who: T::AccountId, expires_at: i64 },
+		/// Intent is requested by some user
+		IntentRequested { who: T::AccountId, intent: Intent },
```

Comment:
We might miss 'IntentId' here for the sake of consistency

Reply:
Yes - I have comment in the extrinsic definition saying this is unused for now. It probably needs a redesign/refactoring anyway if we start to use it